fluent-package/apt/pkgsize-test.sh

```diff
@@ -59,6 +62,11 @@ case ${DISTRIBUTION} in
 esac
 
 set -e
+if [ $PREV_DEB_FOUND -ne 1 ]; then
+	echo "The previous version of deb package is not shipped for ${code_name}, so package size check for ${code_name} is disabled"
+	exit 0
+fi
+
```

Comment:
How about just outputting the new package size if the `$BASE_NAME` file does not exist?
(Looks like adding a new flag `PREV_DEB_FOUND` is not necessary.)
For example, as follows.

```bash

set -e

DEB=$(find $REPOSITORIES_DIR/${DISTRIBUTION}/pool/${CODE_NAME}/${CHANNEL}/f/fluent-package/fluent-package_*${ARCH}.deb | sort -n | tail -1)
CURRENT_SIZE=$(stat -c %s $DEB)
CURRENT_SIZE_MIB=$(echo "scale=2; ${CURRENT_SIZE} / 1024 / 1024" | bc)

if [ ! -e $BASE_NAME ]; then
    echo "OLD: Not found"
    echo "NEW: ${CURRENT_SIZE_MIB} MiB (${CURRENT_SIZE}) : ${DEB}"
    exit 0
fi

PREVIOUS_SIZE=$(stat -c %s $BASE_NAME)
THRESHOLD_SIZE=`echo "$PREVIOUS_SIZE * 1.2" | bc -l | cut -d. -f1`
PREVIOUS_SIZE_MIB=$(echo "scale=2; ${PREVIOUS_SIZE} / 1024 / 1024" | bc)

echo "OLD: ${PREVIOUS_SIZE_MIB} MiB (${PREVIOUS_SIZE}) : ${BASE_NAME}"
echo "NEW: ${CURRENT_SIZE_MIB} MiB (${CURRENT_SIZE}) : ${DEB}"
if [ $CURRENT_SIZE -gt $THRESHOLD_SIZE ]; then
    echo "${DEB} size (${CURRENT_SIZE}) exceeds ${THRESHOLD_SIZE}. Check whether needless file is bundled or not"
    exit 1
fi
```

Reply:
Thanks, it seems better.